client/state/ui/selectors/get-selected-site-id.ts

```diff
@@ -0,0 +1,6 @@
+import { IAppState } from 'calypso/state/types';
+import 'calypso/state/ui/init';
+
+export default function getSelectedSiteId( state: IAppState ): number | null {
```

Comment:
JSDoc description can stay.

Reply:
Is it for the sake of readability here? The inline description doesn't provide much context into the functionality of this selector `Returns the currently selected site ID.`

Reply:
Description can be anything that makes it easy to understand and it can also be used by automatic doc generators.